src/models/document/document.test.ts

```diff
@@ -178,6 +185,35 @@ describe("document model", () => {
     expect(document.content!.tileMap.size).toBe(0);
   });
 
+  it("allows undo and redo", async () => {
+    document.treeMonitor!.enabled = true;
+    const manager = document.treeManagerAPI as Instance<typeof TreeManager>;
+
+    const result = document.addTile("text");
```

Comment:
looks like result is not used so this could be changed to:
```suggestion
    document.addTile("text");
```